UWPHook/Crc32.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace UWPHook
```

Comment:
Would you be willing to add a few comments to this file explaining how the bit wise operations are working here and a little bit of the logic? If you translated the code from another project, a comment stating where does this algorithm came from would be great!

Reply:
I got this code from [this](https://github.com/nullfx/NullFX.CRC) nuget package, actually (at the time I didn't see it). I try replacing this class with said package.

Reply:
Resolved in [e3abf49](https://github.com/BrianLima/UWPHook/pull/50/commits/e3abf499ba304fb45e1d02316df1495bb83a3b4b)

The package I mentioned earlier is .NET Core exclusive, so I went with [Crc32.NET](https://www.nuget.org/packages/Crc32.NET/).